clients/fossid-webapp/src/test/kotlin/FossIdClientNewProjectTest.kt

```diff
@@ -156,7 +152,7 @@ class FossIdClientNewProjectTest : StringSpec({
 
     "Scan status can be queried" {
         service.checkScanStatus("", "", SCAN_CODE) shouldNotBeNull {
-            checkResponse("get scan status", false)
+            checkResponse("get scan status")
```

Comment:
This now changes behavior as `false` (which is not the default) is omitted. Is this intended?

Reply:
Please see new commit for the explanation.